SIL.Media/SIL.Media.csproj

```diff
@@ -10,9 +10,14 @@
     <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
     <UseWindowsForms>true</UseWindowsForms>
     <GenerateResourceUsePreserializedResources>true</GenerateResourceUsePreserializedResources>
+    <Copyright>Copyright © 2010-2023 SIL International</Copyright>
```

Comment:
The Copyright tag is in `/Directory.Build.props` and should be updated there :smile: